### ISSUE TITLE
I got Chrome freeze when access easel.hackclub.com

### ISSUE THREAD

**Issue by nguyengiabach1201** [hackclub/easel#6]
When I enter "How to write a lexer" chapter, it suddenly freeze my browser, which is latest Chrome on Window 11

**Comment by jianmin-chen** [comment]
Hm, is it just freezing up? Have you tried it on a diff browser?

**Comment by nguyengiabach1201** [comment]
Yes, it ran for some seconds and then froze. I had tried on Bing but the result is the same. Yet, it is still ok on my phone:vv
